src/main/resources/templates/macros/report/elements.vm

```diff
@@ -39,6 +39,8 @@
                     </tr>
                   #end
                 </table>
+              #elseif ($step.getDocString())
```

Comment:
can you make this expandable? I guess that would be nice to make this hidden/invisible if the docstring is very long